org.frescobaldi.Frescobaldi.yaml

```diff
@@ -0,0 +1,158 @@
+---
+app-id: org.frescobaldi.Frescobaldi
+
+# Known Flatpak problems:
+# - Documentation browser is not working. See issue https://github.com/frescobaldi/frescobaldi/issues/919 (fixed in master).
+# - Clicking on the print button will crash the application. See issue https://github.com/frescobaldi/frescobaldi/issues/1221
+
+runtime: org.kde.Platform
+runtime-version: '5.13'
+sdk: org.kde.Sdk
+
+command: frescobaldi
+rename-desktop-file: frescobaldi.desktop
+rename-icon: frescobaldi
```

Comment:
Not a blocker but those files should be renamed upstream

Reply:
I'll try to propose the change and if accepted I'll update the manifest for the next release.

Reply:
Hopefully it will be changed before the next release. I'll take care of it as soon as I've done here.

Reply:
For the records, upstream now uses reverse DNS name and ships the metainfo file. It will be available in next version (3.1.2).